conf/application.conf

```diff
@@ -138,6 +138,7 @@ features {
   displayCovid.enabled = true
   missingTraderAddressIntercept.enabled = true
   r17Content.enabled = true
+  directDebitInterrupt.enabled = true
```

Comment:
Can you please also add the feature switch elements for this to allow it to be toggled via the feature switch test-only page